packages/ListBox/src/components/A11y/A11y.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/no-unused-prop-types */
+
+/* A shell component to collect and distribute a11y related props and DOM
+ * attributes to the appropriate (interactive) element
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { RefOf } from "@paprika/helpers";
+
+export default function A11y() {
+  return <></>;
```

Comment:
`return null;` confuses `react-docgen` and this component won't get added to the props tables.